api/controllers/UserController.ts

```diff
@@ -28,10 +29,22 @@ export class UserController {
   @Inject()
   private storageService: StorageService;
 
+  @Get('/:uuid/activity/')
```

Comment:
Nope, doesn't seem to break any existing routes. Maybe routing-controllers handles pattern matching differently than vanilla express? Since `EventController` also has sub-routes that don't break existing ones